packages/invoices/src/InboundShipment/DetailView/api.ts

```diff
@@ -325,10 +325,7 @@ export const useInboundFields = <KeyOfInvoice extends keyof Invoice>(
           id,
         ]);
 
-        queryClient.setQueryData(['invoice', id], {
-          ...previousInbound,
-          ...patch,
-        });
+        queryClient.setQueryData(['invoice', id], patch);
```

Comment:
This is causing the loss of focus - The query data becomes just a partial inbound like `{ id, comment}`, which the whole page reacts to - there's no lines, so the table goes away, there's no other party so the other party input goes away then the status isn't an editable status, so the whole form is disabled causing the loss of focus. I've put it back to what is was previously and it's working fine and updating - but adding items isn't as that uses the `upsert` mutation which includes updating the shipment including the status..!

Reply:
oh! 🤦  thanks